src/sdk/CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ if(TESTING_ENABLE)
     add_executable(db_sdk_test db_sdk_test.cc)
     target_link_libraries(db_sdk_test base_test ${BIN_LIBS} ${THIRD_LIBS})
 
+    add_executable(result_set_sql_test result_set_sql_test.cc)
+    target_link_libraries(result_set_sql_test base_test ${BIN_LIBS} ${THIRD_LIBS})
```

Comment:
unit tests in `src/sdk` should append into list variable `test_list` so coverage will run it

Reply:
tracking in #2576